app/views/schedules/_root_task_fields.html.erb

```diff
@@ -0,0 +1,46 @@
+<div class="nested-fields form-inline">
+  <div class="form-group">
+    <div class="input-group">
+        <% if f.object.id.nil? %>
+            <span class="input-group-addon glyphicon glyphicon-cog">&nbsp;</span>
+        <% else %>
+            <span class="input-group-addon" style="font-weight: bold;">#<%= f.object.id %></span>
+        <% end %>
+        <%= f.label :type, class: 'sr-only' %>
+        <%= f.select :type,
+                     Task.task_options,
+                     {},
+                     class: 'form-control' %>
+        <span class="input-group-addon"></span>
+        <%= f.label :sprout, class: 'sr-only' %>
+        <%= f.select :sprout,
+                     EquipmentProfile.equipment_profile_options,
+                     {include_blank: true, selected: (f.object.sprout.nil?  ? '' : f.object.sprout_name)},
+                     class: 'form-control' %>
+
+        <span class="input-group-btn">
+        <% unless !f.object.schedule.nil? and f.object.id == f.object.schedule.root_task_id %>
```

Comment:
Similarly, since this is only for the creation action and we're limiting this to one Task, you should be able to remove everything in this `unless` block.